.github/workflows/ci.yml

```diff
@@ -217,6 +217,16 @@ jobs:
       - run: ./chain/jsonrpc/build_errors_schema.sh
       - run: git diff --quiet ./chain/jsonrpc/res/rpc_errors_schema.json || exit 1
 
+  cargo_audit:
+    name: "Cargo Audit"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: baptiste0928/cargo-install@21a18ba3bf4a184d1804e8b759930d3471b1c941
+        with:
+          crate: cargo-audit
+      - run: cargo audit -D warnings
```

Comment:
Please add this as a nextest style test while we’re hashing out the justfile questions. Otherwise this is now just another example of a check that only runs in the CI and one that one will most likely not notice failing before waiting ~15 minutes of CI (especially since the CI is always "red" anyway)

Reply:
Hm… OTOH I’m not sure if we should make this a required check (which adding it as a style check would enforce.) One issue with cargo audit checks is that more often than not this check will fail due to changes in the environment (advisories being published) rather than the changes to the PR code, and I’m not sure if we should stop the development project-wide when something like that happens…

Reply:
The CI is no longer always red, since https://github.com/near/nearcore/pull/10218 :) (it really annoyed me too) So this would probably be noticed within a 20 seconds of the job starting, considering the current time to run the action.

I agree with the issues with making this required. TBH I’m not sure either. I think it’d probably make sense to make it required, but will definitely ask on zulip before doing so.

As for the justfile story, TBH I think this will probably not be ready to land before we complete it, considering that #10201 will probably be slow to review, and I’m planning on writing the Justfile tomorrow, whether #10180 has landed or not :) (though I’m hoping it can have landed before and I can submit it in another PR, because with the current timezone difference the number of round-trips would mean I’d make a fourth day in a row working late to get #10180 to land before going OOO tomorrow evening)